test/runnable/testconst.d

```diff
@@ -2806,9 +2806,9 @@ void test6982()
 /************************************/
 // 7038
 
```

Comment:
Just to really nail it down, could you please write this:
```d
static assert(is(S7038 == const S7038)));
```

There should be tests for the other qualifiers as well, notably shared.